src/services/userService.ts

```diff
@@ -13,21 +13,22 @@ export class UserService implements IUserService {
         private readonly alertHelper: IAlertHelper
     ){}
 
-    async createUser(user: User): Promise<void> {
+    async createUser(user: User): Promise<User | undefined> {
         try {
             user.color = defineRandomColor()
             await this.userDatabaseRepository.create(user)
             this.alertHelper.alertSucess("Usuário criado com sucesso!")
+            return user
         } catch(err){
             console.error(err)
             this.alertHelper.alertError("Não foi possível fazer o cadastro.")
         }
     }
 
-    async findUser(user: User): Promise<User | undefined>{
+    async findUser(email: string): Promise<User | undefined>{
```

Comment:
It didn't make sense for this `findUser` function to take the whole user object as a parameter. It only needs the email.